### DISCUSSION TITLE
Welcome to Shadowrocket  Discussions!

### DISCUSSION THREAD

**Discussion by cute** [Shadowrocket/manual#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raza-code** [comment]
Hi 
I want to use Shadowrocket with socks5 but after add ip and port # and click done button nothing appears in bottom of add server

**Reply by cute** [reply]
try to close Shadowrocket and reopen it.
